lib/core/src/lib/form/components/widgets/core/form-field-validator.spec.ts

```diff
@@ -723,9 +723,9 @@ describe('FormFieldValidator', () => {
 
         it('should take into account that max value is in UTC and fail validating value checking the time', () => {
             const maxValueFromActivitiInput = '31-3-2018 12:00 AM';
-            const maxValueSavedInForm = moment(maxValueFromActivitiInput, 'DD-M-YYYY hh:mm A').utc().format();
+            const maxValueSavedInForm = DateFnsUtils.parseDate(maxValueFromActivitiInput, 'DD-M-YYYY hh:mm A').toISOString();
 
-            const localInvalidValue = '2018-3-31 12:01 AM';
```

Comment:
modifying server related value

Reply:
We changed this to accomodate the suggested parseISO() approach used in form-field-model.ts https://github.com/Alfresco/alfresco-ng2-components/pull/8924/commits/e6e0b27b803ba48bebc26729f9185112aceb5e5c#diff-231637c71df1cd50fe2ef9ba319639d7eeb28657b17f552fc979be0b9f508cd0R357.

We were not attempting to change the data here, but parsing it so that it's compatible with .ts logic.

In the previous approach where we were using parseDate(), where we did not need any changes in the test case https://github.com/Alfresco/alfresco-ng2-components/pull/8924/commits/e6e0b27b803ba48bebc26729f9185112aceb5e5c#diff-231637c71df1cd50fe2ef9ba319639d7eeb28657b17f552fc979be0b9f508cd0L357

upon revisiting this I think, the previous approach was better as no modifications were needed in the test case then.

Reply:
`const localInvalidValue = '2018-3-31 12:01 AM';` means invalid date as string, and you are parsing it and converting to iso